pep-0640.rst

```diff
@@ -132,7 +132,8 @@ Backwards Compatibility
 Introducing a new token means there are no backward compatibility concerns.
 No valid syntax changes meaning.
 
-"?" is not considered an identifier, so str.isidentifier() does not change.
+`"?"` is not considered an identifier, so `str.isidentifier()` does not
```

Comment:
```suggestion
``"?"`` is not considered an identifier, so ``str.isidentifier()`` does not
```

Reply:
btw CI linting would catch this: PR https://github.com/python/peps/pull/1570 :)

Reply:
Fixed, thanks :)